### PR TITLE
Initial commit for Nebula65B

### PR DESCRIPTION
**Pull request by yiancar**
## Description
Add Nebula 65B

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/nebula68b/rules.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/nebula68b/rules.mk:16]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/nebula68b/rules.mk [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   comment out to disable the options.
```
Comment (fauxpark) [inline keyboards/nebula68b/rules.mk:8]:
```suggestion
#   change yes to no to disable
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by noroadsleft — COMMENTED** [review]
Your reference images show a 68-key layout with a 2u Backspace, but your `info.json` and layout macro have 69 keys and are set up for a split Backspace. Which is it?

**Comment by yiancar** [comment]
it can do both, split backspace is the most complete.

File: keyboards/nebula68b/info.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "NEBULA68B",
+    "url": "",
+    "maintainer": "yiancar",
+    "layouts": {
+        "LAYOUT_68_ansi": {
```
Comment (noroadsleft) [inline keyboards/nebula68b/info.json:6]:
```suggestion
        "LAYOUT_68_ansi_split_bs": {
```

File: keyboards/nebula68b/info.json [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                {"x":5, "y":0},
+                {"x":6, "y":0},
+                {"x":7, "y":0},
+                {"x":8, "y":0},
+                {"x":9, "y":0},
+                {"x":10, "y":0},
+                {"x":11, "y":0},
+                {"x":12, "y":0},
+                {"x":13, "y":0},
+                {"x":13, "y":0},
```
Comment (noroadsleft) [inline keyboards/nebula68b/info.json:22]: Your keys are overlapping here.

```suggestion
                {"x":13, "y":0},
                {"x":14, "y":0},
```

File: keyboards/nebula68b/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[0] = LAYOUT_68_ansi( /* Base */
```
Comment (noroadsleft) [inline keyboards/nebula68b/keymaps/default/keymap.c:19]:
```suggestion
[0] = LAYOUT_68_ansi_split_bs( /* Base */
```

File: keyboards/nebula68b/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    KC_GESC, KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_DEL, KC_BSPC,  KC_INS,  KC_PGUP,
+    KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_PGDN,
+    KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+    KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,                            KC_UP,
+    KC_LCTL, KC_LGUI, KC_LALT,                KC_SPC,                          KC_RALT, MO(1),   KC_RCTL,                   KC_LEFT, KC_DOWN, KC_RGHT),
+
+[1] = LAYOUT_68_ansi( /* FN */
```
Comment (noroadsleft) [inline keyboards/nebula68b/keymaps/default/keymap.c:26]:
```suggestion
[1] = LAYOUT_68_ansi_split_bs( /* FN */
```

File: keyboards/nebula68b/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[0] = LAYOUT_68_ansi( /* Base */
```
Comment (noroadsleft) [inline keyboards/nebula68b/keymaps/via/keymap.c:19]:
```suggestion
[0] = LAYOUT_68_ansi_split_bs( /* Base */
```

File: keyboards/nebula68b/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    KC_GESC, KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_DEL, KC_BSPC,  KC_INS,  KC_PGUP,
+    KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_PGDN,
+    KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+    KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,                            KC_UP,
+    KC_LCTL, KC_LGUI, KC_LALT,                KC_SPC,                          KC_RALT, MO(1),   KC_RCTL,                   KC_LEFT, KC_DOWN, KC_RGHT),
+
+[1] = LAYOUT_68_ansi( /* FN */
```
Comment (noroadsleft) [inline keyboards/nebula68b/keymaps/via/keymap.c:26]:
```suggestion
[1] = LAYOUT_68_ansi_split_bs( /* FN */
```

File: keyboards/nebula68b/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_DEL,  KC_TRNS, KC_TRNS,
+    KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, RGB_SPD, RGB_SPI, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,
+    KC_VOLU, KC_VOLD, KC_MUTE,                KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS),
+
+[2] = LAYOUT_68_ansi( /* Empty for dynamic keymaps */
```
Comment (noroadsleft) [inline keyboards/nebula68b/keymaps/via/keymap.c:33]:
```suggestion
[2] = LAYOUT_68_ansi_split_bs( /* Empty for dynamic keymaps */
```

File: keyboards/nebula68b/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS,                KC_TRNS,                               KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS),
+
+[3] = LAYOUT_68_ansi( /* Empty for dynamic keymaps */
```
Comment (noroadsleft) [inline keyboards/nebula68b/keymaps/via/keymap.c:40]:
```suggestion
[3] = LAYOUT_68_ansi_split_bs( /* Empty for dynamic keymaps */
```

File: keyboards/nebula68b/nebula68b.h [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+ */
+#pragma once
+
+#define XXX KC_NO
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+
+#define LAYOUT_68_ansi( \
```
Comment (noroadsleft) [inline keyboards/nebula68b/nebula68b.h:24]:
```suggestion
#define LAYOUT_68_ansi_split_bs( \
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
* renames `LAYOUT_68_ansi` to `LAYOUT_68_ansi_split_bs`

**Comment by yiancar** [comment]
Cheers thank you,
Sorry I didnt realize 68% split bc was in the standard layouts:D

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**